pkg/plugins/runtime/gateway/gateway_route_generator_test.go

```diff
@@ -2123,15 +2218,10 @@ conf:
 
 	Context("with a TCP gateway", func() {
 		DescribeTable("generating xDS resources",
-			func(goldenFileName string, fixtureResources ...string) {
+			func(goldenFileName string, args ...interface{}) {
 				// given
-				// #nosec G404 -- used just for tests
-				r := rand.New(rand.NewSource(GinkgoRandomSeed()))
-				r.Shuffle(len(fixtureResources), func(i, j int) {
-					fixtureResources[i], fixtureResources[j] = fixtureResources[j], fixtureResources[i]
-				})
-				for _, resource := range fixtureResources {
-					Expect(StoreInlineFixture(rt, []byte(resource))).To(Succeed())
+				for _, arg := range args {
+					handleArg(arg)
 				}
```

Comment:
Ok, I don't understand why it was necessary before and is not anymore. Could you explain it to me?

Reply:
Ok, I get it now

Reply:
I've changed a test to handle the option to remove default policies. In some cases, we were testing Timeout policy and they were overridden by MeshTimeout.